internal/health/health.go

```diff
@@ -0,0 +1,186 @@
+package health
```

Comment:
it would be good to have some minimal tests, e.g. for not ready it can be sth like:
```go
func TestServeHTTP(t *testing.T) {
	// given
	checker := &Checker{
		config: &config.Config{},
	}
	expectedStatus := checker.GetStatus()

	req, err := http.NewRequest("GET", "/", nil)
	require.NoError(t, err)
	rr := httptest.NewRecorder()

	// when
	checker.ServeHTTP(rr, req)

	// then
	assert.Equal(t, http.StatusServiceUnavailable, rr.Code)
	assert.Equal(t, "application/json", rr.Header().Get("Content-Type"))

	var resp Status
	err = json.Unmarshal(rr.Body.Bytes(), &resp)
	require.NoError(t, err)

	assert.Equal(t, BotkubeStatusUnhealthy, resp.Botkube.Status)
	assert.Equal(t, resp.Botkube.Status, expectedStatus.Botkube.Status)
}

```